react/modules/gaEvents.ts

```diff
@@ -118,3 +119,36 @@ export function selectPromotion(eventData: PixelMessage['data']) {
 
   updateEcommerce(eventName, data)
 }
+
+export function addToCart(eventData: PixelMessage['data']) {
+  if (!shouldMergeUAEvents()) return
+
+  const eventName = 'add_to_cart'
+
+  const {
+    currency,
+    items: [item],
+  } = eventData
+
+  const productName = getProductNameWithoutVariant(item.name, item.skuName)
+  const formattedPrice =
+    item.priceIsInt === true ? item.price / 100 : item.price
+
+  const data = {
+    currency,
+    value: formattedPrice,
```

Comment:
Assuming there could be multiple items being added to the cart, as I mentioned in a previous comment, I'm assuming we need to update this `value` to be the sum of all the items' prices.